platform/kube/inject/inject_test.go

```diff
@@ -166,23 +164,18 @@ func TestIntoResourceFile(t *testing.T) {
 			Policy:     InjectionPolicyEnabled,
 			Namespaces: []string{v1.NamespaceAll},
 			Params: Params{
-				InitImage:         InitImageName(unitTestHub, unitTestTag, c.debugMode),
-				ProxyImage:        ProxyImageName(unitTestHub, unitTestTag, c.debugMode),
-				ImagePullPolicy:   "IfNotPresent",
-				Verbosity:         DefaultVerbosity,
-				SidecarProxyUID:   DefaultSidecarProxyUID,
-				Version:           "12345678",
-				EnableCoreDump:    c.enableCoreDump,
-				Mesh:              &mesh,
-				MeshConfigMapName: "istio",
-				DebugMode:         c.debugMode,
+				InitImage:       InitImageName(unitTestHub, unitTestTag, c.debugMode),
+				ProxyImage:      ProxyImageName(unitTestHub, unitTestTag, c.debugMode),
+				ImagePullPolicy: "IfNotPresent",
+				Verbosity:       DefaultVerbosity,
+				SidecarProxyUID: DefaultSidecarProxyUID,
+				Version:         "12345678",
+				EnableCoreDump:  c.enableCoreDump,
+				Mesh:            &mesh,
+				DebugMode:       c.debugMode,
```

Comment:
wtb github diff -w (ignore whitespaces) - this is a lot harder to read as a diff than it should be